frontend/tests/blocks.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { expect } from "@playwright/test";
+import { test } from "./util/common";
+import { USERS } from "./util/user";
+import { realmSetup, realms } from "./util/realm";
+import { addBlock, editBlock, testBlocks } from "./util/blocks";
```

Comment:
The test blocks I would move into this file. It's just fixed data again, not something that's a general utility. We only use it in this file right?